controllers/mattermost/mattermost/mattermost_test.go

```diff
@@ -265,10 +266,12 @@ func TestCheckMattermost(t *testing.T) {
 
 		err = reconciler.Client.Update(context.TODO(), modified)
 		require.NoError(t, err)
+		recStatus, err = reconciler.checkMattermostDeployment(mm, dbInfo, fileStoreInfo, currentMMStatus, logger)
```

Comment:
Lets move it after we assert that job was created, but before we update it with proper status (around line 286). We also should assert it does not error and the `ResourceReady` is `false`.